participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/UserRegistrationController.java

```diff
@@ -50,12 +50,13 @@
   public ResponseEntity<UserRegistrationResponse> registerUser(
       @Valid @RequestBody UserRegistrationForm user,
       @RequestHeader("appId") String appId,
+      @RequestHeader String appName,
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.